.github/workflows/gh-pages.yml

```diff
@@ -21,11 +21,11 @@ jobs:
       - name: Install and Build 🔧
         run: |
           npm install
-          npm run preinstall
+          npm run build:miden
           npm run build
           npx webpack
       - name: Deploy to GitHub Pages
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: build
\ No newline at end of file
+          publish_dir: build
```

Comment:
I would keep the newline at the end of this file

Reply:
done